src/pages/team/languages.json

```diff
@@ -183,7 +183,7 @@
         "email": "kirs@apache.org",
         "gitUrl": "https://github.com/CalvinKirs",
         "githubId": "Calvin Kirs",
-        "name": "Calvin Kirs"
+        "name": "CalvinKirs"
```

Comment:
it should be the ID field does not allow spaces

Reply:
ok, fixed.